app/helpers/patient_query_helper.rb

```diff
@@ -550,7 +550,7 @@ def linelist(patients, workflow, tab)
                                'patients.last_assessment_reminder_sent, patients.preferred_contact_time, patients.extended_isolation, '\
                                'patients.latest_fever_or_fever_reducer_at, patients.latest_positive_lab_at, patients.negative_lab_count, '\
                                'patients.head_of_household, jurisdictions.name AS jurisdiction_name, jurisdictions.path AS jurisdiction_path, '\
-                               'jurisdictions.id AS jurisdiction_id')
+                               'jurisdictions.id AS jurisdiction_id', 'patients.latest_assessment_symptomatic')
```

Comment:
sorry for being nitpicky, but can we add `'patients.latest_assessment_symptomatic'` right after `'patients.latest_assessment_at'` on line 547?